src/main/java/org/example/nikishma/Primenumber.java

```diff
@@ -0,0 +1,31 @@
+package org.example.nikishma;
+import java.util.Scanner;
+
+public class Primenumber {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter an integer: ");
+        int num = scanner.nextInt();
+
+        boolean isPrime = true;
+        for (int i = 2; i <= num / 2; i++) {
+            if (num % i == 0) {
+                isPrime = false;
```

Comment:
Once we know that the number is not prime, we can exit the loop. User `break;` statement